### DISCUSSION TITLE
Kia Connect for Australia

### DISCUSSION THREAD

**Discussion by grouts112** [Hyundai-Kia-Connect/kia_uvo#465] [Ideas]
I have just picked up a new Kia Niro MY23 in Australia, I have it working nicely on the Kia connect app and would love to do what I can to get Australian Kia Connect into this integration for HA.
I am not a programmer, but can follow instructions to gather any data required.

**Comment by cdnninja** [comment]
This thread is where currently discussed.   We need someone to crack the stamps and document the api.  

https://github.com/Hyundai-Kia-Connect/kia_uvo/discussions/462
